plugins/BEdita/API/tests/TestCase/Controller/AppControllerTest.php

```diff
@@ -96,20 +250,49 @@ public function contentTypeProvider()
      * @dataProvider contentTypeProvider
      * @covers \BEdita\API\Controller\Component\JsonApiComponent::startup()
      * @covers \BEdita\API\Controller\Component\JsonApiComponent::beforeRender()
+     * @covers \BEdita\API\Error\ExceptionRenderer::render()
      */
-    public function testContentType($expectedCode, $expectedContentType, $accept, array $config = null)
+    public function testContentType($expectedCode, $expectedContentType, $accept, $endpoint, array $config = null)
```

Comment:
Are we testing too many things? 
Should we have different tests for content type negotiation and generic error responses?

Don't have a solution/proposal but we may probably split this in simpler test methods